awx/main/utils/licensing.py

```diff
@@ -382,8 +382,15 @@ def validate(self):
 
         current_instances = Host.objects.active_count()
         license_date = int(attrs.get('license_date', 0) or 0)
-        automated_instances = HostMetric.active_objects.count()
-        first_host = HostMetric.active_objects.only('first_automation').order_by('first_automation').first()
+
+        model = getattr(settings, 'SUBSCRIPTION_USAGE_MODEL', '')
+        if model == 'unique_managed_hosts':
+            automated_instances = HostMetric.active_objects.count()
+            first_host = HostMetric.active_objects.only('first_automation').order_by('first_automation').first()
+        else:
+            automated_instances = HostMetric.objects.count()
```

Comment:
Might be more understandable to put None here. But it really doesn't matter what we set here, since UI should hide the compliance/consumption related fields if model != 'unique_managed_hosts'

Reply:
as it doesn't matter, I'll not change the current behaviour for AWX and keep it